build_posix/configure.ac.in

```diff
@@ -140,26 +140,14 @@ if test "$wt_cv_enable_strict" = "yes"; then
 	AM_CFLAGS="$AM_CFLAGS $wt_cv_strict_warnings"
 fi
 
-# Java and Python APIs
-if test "$wt_cv_enable_java" = "yes" -o "$wt_cv_enable_python" = "yes"; then
+# Python APIs
```

Comment:
nit: should say `Python API` (2 places in this file).

Reply:
Fixed.